### PR TITLE
0.13: Primitive shapes

### PR DESCRIPTION
**Pull request by Jondolf**
For the 0.13 announcement post: #891

This is WIP and missing some parts. I wrote the introduction and will add meshing soon. I left sections where @RobWalt and @NiseVoid can add their own sections on gizmos and bounding volumes respectively.

### CODE REVIEW

**Comment by alice-i-cecile** [comment]
This is looking pretty good to me: what more did you want to do here?

**Comment by Jondolf** [comment]
It's missing the gizmo section. @RobWalt has a PR on my branch here https://github.com/Jondolf/bevy-website/pull/1, but it still has TODOs

**Comment by RobWalt** [comment]
I'm still waiting on the dedicated gizmo rendering example to be merged to point to the example for users to try it themselves. I'll finish up the rest of the PR into this branch tomorrow 👍

**Comment by alice-i-cecile** [comment]
PR in question: https://github.com/bevyengine/bevy/pull/11697

**Comment by Jondolf** [comment]
This should now be ready! I added links for *every* type here, let me know if it's overkill

File: content/news/2024-02-03-bevy-0.13/index.md [hunk 1]
```diff
@@ -31,9 +31,241 @@ Since our last release a few months ago we've added a _ton_ of new features, bug
 
 ## Primitive Shapes
 
-<div class="release-feature-authors">authors: @TODO</div>
+<div class="release-feature-authors">authors: @Jondolf, @NiseVoid</div>
```
Comment (alice-i-cecile) [inline content/news/2024-02-03-bevy-0.13/index.md:34]:
```suggestion
<div class="release-feature-authors">authors: @Jondolf, @NiseVoid, @aevyrie</div>
```

**Pull request merged**